assets/prefabs/structures/dungeon/dungeon3x3StoneCorridor/SandCorridor.prefab

```diff
@@ -0,0 +1,38 @@
+{
+    "StructureTemplate": {
+        "type": "GooeysQuests:dungeon3x3StoneCorridor",
+        "spawnChance": 1000
```

Comment:
This is good for testing but should probably be turned down for "live" play :-)

I might suggest `10` or `20` to make it somewhat rare